src/components/AvatarWithIndicator.js

```diff
@@ -0,0 +1,51 @@
+import React, {memo} from 'react';
+import {View, StyleSheet} from 'react-native';
+import PropTypes from 'prop-types';
+import Avatar from './Avatar';
+import styles from '../styles/styles';
+
+const propTypes = {
+    // Is user active?
+    isActive: PropTypes.bool,
+    source: PropTypes.string.isRequired,
```

Comment:
All propTypes should have some kind of comment.

Reply:
Alright